tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -1274,7 +1274,7 @@ def test_assert_with_exception(form, wast_tempfile, wasm_tempfile, aot_tempfile,
             else:
                 raise Exception("unrecognized form '%s...'" % form[0:40])
     except Exception as e:
-        traceback.print_exc()
+        traceback.print_exc(file=sys.stdout)
```

Comment:
hi @yamt , could you clarify what is the benefit of this change? Whereas I understand what it does, it's not quite clear why do we want to change it?